### ISSUE TITLE
Would be great to map Sparkleshare to a branch

### ISSUE THREAD

**Issue by ph-hs** [hbons/SparkleShare#1659]
The new project wizard asks for the "Address" and the "Remote path". It would be great if it were possible to specify a branch (that has a default behavior as now). 

If the branch name could make it into the working directory name it would be even better :)

**Comment by hbons** [comment]
I don't like the idea of exposing this in the UI, but I can see specifying branches being useful in Invites and Plugins.

**Comment by ph-hs** [comment]
I'll be guided by your expertise (and can always follow with another feature request).

**Comment by karazy** [comment]
I would also love this feature. Are there any new developments regarding this?

**Comment by hbons** [comment]
I changed my mind on this. I believe this can be very useful for code hosting places like GitHub.

**Comment by yisraeldov** [comment]
@hbons Yes, the usecase I see for this is working with people who are not very git savy. But we wouldn't want them to be able to push to master.
